es-ds-docs/pages/molecules/rating.vue

```diff
@@ -26,6 +20,11 @@ onMounted(async () => {
         $prism.highlight();
     }
 });
+
+const rating1Val = ref(0);
+watch(rating1Val, () => {
+    alert(rating1Val.value);
```

Comment:
Add  `// eslint-disable-next-line no-alert` to the previous line to resolve the warning

Reply:
I talked to Eric who suggested removing the alert and displaying the value as text underneath instead